sites/all/modules/custom/da_malik/da_malik.install

```diff
@@ -10,8 +10,9 @@ function da_malik_enable() {
 	variable_set('site_frontpage', $homepage);
 	
 	/*enabling theme 2 and setting it as the default*/
-	theme_enable(array('debuga_theme'));
-	variable_set('theme_default', 'debuga_theme');
+	$debugAcademyThemeTwo = 'debuga_theme2';
```

Comment:
namingLikeThis is usually used for class methods (speaking from PHP). For regular variables $debug_academy_theme_two would be better. Just an FYI